tests/handlers/test_wsgi.py

```diff
@@ -0,0 +1,34 @@
+from unittest import mock
+
+from django.core.handlers.wsgi import WSGIHandler
+from django.http import FileResponse
+from django.test import SimpleTestCase
+from django.test.client import FakePayload
+
+
+class DummyFileResponse(FileResponse):
+    def __init__(self, _, value):
+        super().__init__()
+        self.value = value
+
+
+class TestWsgiHandler(SimpleTestCase):
+    @classmethod
+    def _get_suspicious_environ(cls):
+        payload = FakePayload('a=1&a=2;a=3\r\n')
+        return {
+            'REQUEST_METHOD': 'POST',
+            'CONTENT_TYPE': 'application/x-www-form-urlencoded',
+            'CONTENT_LENGTH': len(payload),
+            'wsgi.input': payload,
+            'SERVER_NAME': 'test',
+            'SERVER_PORT': '8000',
+            'wsgi.file_wrapper': lambda *args: DummyFileResponse(None, args[1] + 1),
+        }
+
+    def test_wsgi_block_size_passing(self):
+        sample_response = FileResponse()
+        sample_response.file_to_stream = iter(['sample_response'])
+        with mock.patch.object(WSGIHandler, 'get_response', return_value=sample_response):
+            response = WSGIHandler()(self._get_suspicious_environ(), lambda *a, **k: None)
+        assert response.value == FileResponse.block_size + 1
```

Comment:
Use `self.assertEqual()` instead of `assert ... == ...` .

Reply:
Thanks :) Done. Please take a look at this PR :)